agent/agent_test.go

```diff
@@ -707,22 +734,61 @@ func (s *AgentSuite) TestEndTaskResponse() {
 }
 
 func (s *AgentSuite) TestOOMTracker() {
-	s.setupRunTask(defaultProjYml)
-	s.tc.project.OomTracker = true
+	projYml := `
+oom_tracker: true
+buildvariants:
+- name: mock_build_variant
```

Comment:
mega nit: since you indent with one space in the other blocks, should also indent here (although YAML generally has 2 or 4 spaces for indentation, not 1).